libctru/source/applets/miiselector.c

```diff
@@ -0,0 +1,42 @@
+#include <3ds/types.h>
+#include <3ds/services/apt.h>
+
+#include <3ds/applets/miiselector.h>
+
+Result miiSelectorLaunch(MiiSelectorContext *ctx)
```

Comment:
I wasn't really happy with the `union` too; this is definitely a better approach. Thanks for the input, will implement!